test/test_pruning_op.py

```diff
@@ -0,0 +1,56 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import hypothesis.strategies as st
+from hypothesis import given
+import numpy as np
+import torch
+from torch.testing._internal.common_utils import TestCase
+
+class PruningOpTest(TestCase):
+
+    def _test_rowwise_prune_op(self, embedding_rows, embedding_dims, indices_type):
+        embedding_weights = torch.from_numpy((np.random.random_sample((embedding_rows, embedding_dims)) + 1).astype(np.float32))
+        mask = torch.from_numpy(np.random.choice(
```

Comment:
Should we modify the test to show how the mask can be generated given indicator and threshold values (which can be random values in the test)?

Reply:
SG. Done